app/src/main/java/edu/akelael/comics/model/ComicListModel.java

```diff
@@ -0,0 +1,48 @@
+package edu.akelael.comics.model;
+
+import java.util.HashMap;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+public class ComicListModel implements IComicListModel {
+    Server server;
+    Retrofit retrofit;
+    HashMap<String, String> mMap;
+
+    @Override
+    public void initializeApiConnection(){
+        Retrofit.Builder retrofitBuilder;
+        retrofitBuilder = new Retrofit.Builder();
+        retrofitBuilder.baseUrl(server.baseUrl);
+        retrofitBuilder.addConverterFactory(GsonConverterFactory.create());
+        retrofit = retrofitBuilder.build();
+        mMap = new HashMap<>();
+        mMap.put("ts", "ts");
+        mMap.put("apikey", "apikey");
+        mMap.put("hash", "hash");
+    }
+
+    @Override
+    public void getComics(final IComicListModelCallbacks listener){
+        server = retrofit.create(Server.class);
```

Comment:
IMO here the method has its own purpose that is "get the comics". In fact, for me it's a candidate to be called from another piece of code, so is better to have it in there with its responsability. 
I think it's not clear to put this on the "initializeApiConnection()" cause also semantically it has other meaning.